target/linux/ramips/dts/HC5611.dts

```diff
@@ -0,0 +1,126 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hiwifi,hc5611", "mediatek,mt7628an-soc";
+	model = "HiWiFi HC5611";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		system {
+			label = "hc5611:green:system";
+			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		power-led {
+			gpio-export,name = "power-led";
+			gpio-export,output = <1>;
+			gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "i2c", "refclk", "wled_an";
```

Comment:
You are not using any of the pins covered by these groups as GPIO. It has to be `"spi_cs1", "wdt"`.

That the red:power LED uses the pin of the `spi_cs1` group looks rather strange. Are you sure it is correct?

Reply:
HC5611.dts just be modified from HC5661A.dts, I can't confirm this.

Reply:
If the buttons and LEDs are still working, my proposed pinmux is correct. I had a look at the datasheet to check which GPIO belongs to which pinmux group.